auto-instrumentation/okhttp/okhttp-3.0/testing/build.gradle.kts

```diff
@@ -4,8 +4,10 @@ plugins {
 }
 
 dependencies {
+    implementation(platform(project(":dependencyManagement")))
+
     byteBuddy(project(":auto-instrumentation:okhttp:okhttp-3.0:agent"))
     implementation(project(":auto-instrumentation:okhttp:okhttp-3.0:library"))
-    implementation("com.squareup.okhttp3:okhttp:4.11.0")
-    androidTestImplementation("com.squareup.okhttp3:mockwebserver:4.11.0")
+    implementation("com.squareup.okhttp3:okhttp")
+    androidTestImplementation("com.squareup.okhttp3:mockwebserver:${rootProject.extra["okhttpVersion"]}")
```

Comment:
Isn't this artifact covered by the okhttp bom defined in `dependencyManagement`?

Reply:
Never mind! I just noticed you mentioned that these `androidTestImplementation` types of dependencies don't seem to work well with BOMs.